build.gradle.kts

```diff
@@ -1,9 +1,14 @@
+import io.gitlab.arturbosch.detekt.Detekt
 import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
 
 plugins {
     kotlin("multiplatform") version "1.6.10"
     id("maven-publish")
+    id("java-library")
+    id("signing")
     id("pl.allegro.tech.build.axion-release") version "1.13.6"
+    id("io.gitlab.arturbosch.detekt") version "1.19.0"
```

Comment:
maybe we should move version to another file? 

Dependencies or something,  and then use  Versions.detekt